TrafficCapture/gradlew

```diff
@@ -241,4 +241,11 @@ eval "set -- $(
         tr '\n' ' '
     )" '"$@"'
 
+# Check if OS_MIGRATIONS_GRADLE_SCAN_TOS_AGREE_AND_ENABLED exists, and if --scan is not in the arguments and --no-scan is not provided
+if [ -n "${OS_MIGRATIONS_GRADLE_SCAN_TOS_AGREE_AND_ENABLED+set}" ] && ! printf '%s\n' "$@" | grep -q -- '--scan' && ! printf '%s\n' "$@" | grep -q -- '--no-scan'; then
+    echo "Enabling Gradle Scan based on environment variable OS_MIGRATIONS_GRADLE_SCAN_TOS_AGREE_AND_ENABLED"
+    set -- "$@" '--scan'  # Append --scan to the arguments
+fi
```

Comment:
This file is autogenerated.  Whenever we upgrade gradle and/or rerun the wrapper task, this will be recreated, removing these lines.  If you could do this within the build.gradle files, the change would be more durable and easier to maintain.

Reply:
I agree, but i had not found a way to do this inside build.gradle hence it's inclusion here